### DISCUSSION TITLE
复杂表头二级或三级头有重复的情况，会导致其中一项的数据为null

### DISCUSSION THREAD

**Discussion by Bertkahn** [alibaba/easyexcel#3732] [General]
@ApiModelProperty("培训人员数：学生")
    @ExcelProperty({"培训人员数", "学生"})
    private Long trainingStudentCount;

    
    @ApiModelProperty("培训人员数：教师")
    @ExcelProperty({"培训人员数", "教师"})
    private Long trainingTeacherCount;

   
    @ApiModelProperty("培训人员数：其他")
    @ExcelProperty({"培训人员数", "其他"})
    private Long trainingOtherCount;

 @ApiModelProperty("发明专利：教师")
    @ExcelProperty({"发明专利", "教师"})
    private Long inventTeacherCount;

   
    @ApiModelProperty("发明专利：学生")
    @ExcelProperty({"发明专利", "学生"})
    private Long inventStudentCount;

上述代码二级表头存在两个学生和两个教师，第一个的学生和教师会在导入的字段为null的情况

**Comment by Jasonyou-boy** [comment]
标记
